runelite-client/src/main/java/net/runelite/client/plugins/skillcalculator/skills/MiningAction.java

```diff
@@ -65,4 +73,10 @@
 	private final int level;
 	private final float xp;
 	private final int icon;
+
+	@Override
+	public boolean isMembers(final ItemManager itemManager)
```

Comment:
During the course of reviewing this PR, I've realized this class can be reimplemented to `implements ItemSkillAction`, and that'll remove the need for this override. The only items I see needing overrides for their names are `BARRONITE_SHARDS` and `GEM_ROCKS`.